bricks/config.rb

```diff
@@ -131,4 +131,3 @@ def custom_config(config)
 
 # Enable port forwarding from guest(s) to host machine, syntax is: { 80 => 8080 }, auto correction is enabled by default.
 $forwarded_ports = { 8443 => 8443 }
-$update_channel='beta'
```

Comment:
What does this do?

Reply:
It was overwriting the `$update_channel='stable'` change in `Vagrantfile`. Basically used to fill out the template `https://${update_channel}.core-os.net/...` 

The `$update_channel` var should be in Vagrantfile, since that's were we configure the box url.